File: cpp/bc2533.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	int fSo, sSo, nCa, fVal, sVal;
	double res;
	
	while(cin >> nCa)
	{
		fSo = sSo = 0;
		
		for(int i = 0; i < nCa; i++)
		{
			cin >> fVal >> sVal;
			
			fSo += fVal * sVal;
			sSo += sVal;
		}
		
		res = (double) fSo / (sSo * 100);
		cout << fixed << setprecision(4) << res << endl;
	}
}

File: python/bc2533.py
while True:
    try:
        fSo = 0
        sSo = 0

        nCa = int(input())

        for c in range(0, nCa):
            cVal = input().split(" ")

            fVal = int(cVal[0])
            sVal = int(cVal[1])

            fSo += fVal * sVal
            sSo += sVal

        res = float(fSo / (sSo*100))

        print(f"{res:.4f}")
    except EOFError:
        break
